src/builder.js

```diff
@@ -313,8 +322,14 @@ export default function builder() {
         data['HTTP_COOKIE'] = encodeCookie(err.cookies);
       }
 
+      const breadcrumbs = {
+        'enabled': breadcrumbsEnabled(),
+        'trail': self.breadcrumbs.slice(0),
```

Comment:
What does 0th slice do?

Reply:
It's the same as calling `array.slice()`. It duplicates the array object so that additional breadcrumbs aren't added to it. I think I'll remove the `0` argument and make it implicit.

Reply:
😢